ROMFS/px4fmu_common/init.d/airframes/1102_tailsitter_duo_sih.hil

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+#
+# @name SIH Tailsitter Duo
+#
+# @type Simulation
+# @class VTOL
+#
+# @output MAIN1 motor right
+# @output MAIN2 motor left
+# @output MAIN5 elevon right
+# @output MAIN6 elevon left
+#
+# @maintainer Romain Chiappinelli <romain.chiap@gmail.com>
+#
+# @board px4_fmu-v2 exclude
+#
+
+. ${R}etc/init.d/rc.vtol_defaults
+
+
+param set-default VT_ELEV_MC_LOCK 0
+param set-default VT_MOT_COUNT 2
+param set-default VT_TYPE 0
+param set-default VT_FW_DIFTHR_EN 1
+param set-default VT_FW_DIFTHR_SC 0.3
+param set-default MPC_MAN_Y_MAX 90
+
+param set-default MAV_TYPE 19
+set MAV_TYPE 19
+set MIXER vtol_tailsitter_duo
+
+set PWM_OUT 1234
+
+# set SYS_HITL to 2 to start the SIH and avoid sensors startup
+param set-default SYS_HITL 2
+
+# disable some checks to allow to fly:
+# - with usb
+param set-default CBRK_USB_CHK 197848
+# - without real battery
+param set-default CBRK_SUPPLY_CHK 894281
+# - without safety switch
+param set-default COM_PREARM_MODE 0
+param set-default CBRK_IO_SAFETY 22027
+
+param set-default BAT_N_CELLS 3
+
+param set SIH_T_MAX 2.0
+param set SIH_Q_MAX 0.0165
+param set SIH_MASS 0.2
+param set SIH_IXX 0.00354 	# IXX and IZZ are inverted from the thesis as the body frame is pitched by 90 deg
+param set SIH_IYY 0.000625
+param set SIH_IZZ 0.00300
+param set SIH_IXZ 0.0
+param set SIH_KDV 0.2
+param set SIH_L_ROLL 0.145
+
+param set SIH_VEHICLE_TYPE 2 	# sih as tailsitter
```

Comment:
Can  you update the parameter metadata as well?

Reply:
I ran `make parameters_metadata`, added, committed, and pushed. Is that enough?

I think they are generated inside ./build which is not committed.

Reply:
Sorry, what I meant was to add tailsitter here: https://github.com/PX4/PX4-Autopilot/blob/c60f0934e94a401ca1d13b181c76143114e5f510/src/modules/sih/sih_params.c#L443